### ISSUE TITLE
French translations

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CMU-CREATE-Lab/create-lab-visual-programmer#150]
```
I would be interested in contributing french translations of the UI texts (if 
not already done). It seems that the code already uses resource bundle loading 
for messages, so there should not be any code change required. I will try with 
adding _fr properties files in the classpath.

Tell me how to proceed for contributing and maintaining the files. You could 
pull them from a github repo of mine or I could even be added as committer of 
your project.

Note that I will be able to test them only for the Hummingbird Kit (just 
ordered). Unfortunately, there is no way to start the application without 
device connected.

BTW, are there any developer resources for visual programmer? I mean forum or 
wiki? 

```

Original issue reported on code.google.com by `laurent....@gmail.com` on 24 Apr 2013 at 8:11
- Blocked on: #120
